logistics.md

```diff
@@ -0,0 +1,144 @@
+
+{% collapse Mattermost %}
+* Generic channels: 
+  * [HBM-hackathon](https://mattermost.brainhack.org/brainhack/channels/hbm-hackathon)
+  * [General](https://mattermost.brainhack.org/brainhack/channels/general)
+  * [Town Square](https://mattermost.brainhack.org/brainhack/channels/town-square)
+* Project channel: Each project will have its own channel
+* Help desk: We will have a set of topic and programming language specific channels where you can ping volunteers for help.
+{% endcollapse %}
+
+{% collapse Zoom/Jitsi %}
+Each project will be using Zoom or Jitsi for their individual calls.
+{% endcollapse %} 
+{% endaccordion %}
+
+
+#### HackTrack
+{% accordion HackTrack %} 
+As with previous OHBM hackathons, we've developed a project template that we would like project leads to use for submitting 
+their project proposals.  
+{% collapse Project submission %}
+Each project is allocated to a project helper who will follow the project from submission until the end of the hackathon.
+To make sure that project helpers have enough time to review project submissions, we're setting two deadlines: 
+1. 2 weeks before the hackathon: submit a draft for project helpers to give feedback on
+2. 1 week before hackathon: hard deadline
+
+[**Project checklist**](https://docs.google.com/spreadsheets/d/1SY-Hnhc-bPX0RA47icVukUVVJ-BrX9ykPFQUBY-U1go/edit#gid=0): this is a spreadsheet to help the project helpers making sure that each project is on track and submitted everything.
+
+[**Github issue template**](https://github.com/ohbm/hackathon2020/blob/master/.github/ISSUE_TEMPLATE/hackathon-project-template.md): This will be the way to submit hackathon projects. It comes with a checklist to make sure that project leaders and participants can start in the best conditions.
+
+Once a project has been submitted the project helper can tag it using issue labels to make it easier to search amongst the different projects.
+
+{% endcollapse %}
+
+{% endaccordion %}
+
+
+#### TrainTrack 
+The TrainTrack will cover topics ranging from good code practices, code and data version control, high performance computing, 
+data visualization, and machine learning. Each session will have its own computing requirements, but at a baseline, 
+we strongly recommend that participants make sure that they have the following general computing requirements satisfied. 
+We will have a virtual help desk available to help participants with installation issues. 
+
+{% accordion TrainTrack %}
+{% collapse General computing requirements %}
+In addition to some software that we will list in the subsequent section, these are the general computing requirements: 
+1. Administrative access to your computer such that you can install software on your own. 
+2. If you're using Windows, the required version is Windows 10. 
+{% endcollapse %}
+
+{% collapse Required software %}
+Along with the platforms that we will be using for virtual TrainTrack presentations and communication, we ask that you 
```

Comment:
Maybe  we will need to mention that this is i they want to attend the whole train track and that the requirement may vary.

Can be updated later.

Reply:
That's true - we could modify the installation verification script as well based on a session(s), but I do think for now we could leave it as is for simplicity